### DISCUSSION TITLE
Public Preview Release

### DISCUSSION THREAD

**Discussion by jaymo001** [feathr-ai/feathr#39] [Announcements]
Hello world!
We are excited to announce Feathr public preview release v0.1.0.
Please try it out and join our community. We are looking forward to hearing feedback from you!

<hr /><em>This discussion was created from the release <a href='https://github.com/linkedin/feathr/releases/tag/v0.1.0'>Public Preview Release</a>.</em>

**Comment by JamesUntamed** [comment]
I have FEATHR UI version 0.9.0 installed in Azure. How do I upgrade to the latest version without doing a full blown redeployment? Thanks!
